src/ws-dropdown/dropdown-input.js

```diff
@@ -35,7 +35,7 @@ export class DropdownInput extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      value: props.value
+      value: props.value && props.value.value
```

Comment:
is it intended to have a possible 'undefined' as a state value ? Seems a bit unsafe

Reply:
Yes it's intended to have possibly an undefined as you can include a dropdown without having a default value or a previously persisted value. 
Would a ternary be better for you? Like
```js
this.state = {
  value: props.value ? props.value.value : null
}
```